scripts/update-bower.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Update all bower packages
+bower update
+
+# Link bootstrap3 theme to bootstrap
+pushd nikola/data/themes/bootstrap3/assets/js/
+ln -sf ../../../../../../bower_components/bootstrap/dist/js/*js .
+rm npm.js
+git add *js
```

Comment:
`git add *` can be problematic.  use `git add .` to add everything in the current working directory.